Makefile

```diff
@@ -7,26 +7,19 @@ include versioning.mk
 
 build: docker-build
 push: docker-push
-install: kube-install
-uninstall: kube-delete
-upgrade: kube-update
 
 docker-build:
 	docker build ${DOCKER_BUILD_FLAGS} -t ${IMAGE} rootfs
 	docker tag ${IMAGE} ${MUTABLE_IMAGE}
 
-kube-delete:
-	-kubectl delete -f manifests/deis-logger-fluentd-daemon.tmp.yaml
-
-kube-install: update-manifests
-	kubectl create -f manifests/deis-logger-fluentd-daemon.tmp.yaml
+test: docker-build
+	docker run ${IMAGE} /bin/bash -c "cd /opt/fluentd/deis-output && rake test"
 
-kube-update: update-manifests
-	kubectl delete -f manifests/deis-logger-fluentd-daemon.tmp.yaml
-	kubectl create -f manifests/deis-logger-fluentd-daemon.tmp.yaml
+install:
```

Comment:
no install performs a `--install` with the helm upgrade command. but upgrade does not. one if for installing a fresh chart the other is to update the daemonset